src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -39,6 +45,8 @@ public PanelControl(UI ui, Preferences prefs) {
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
         ui.registerEvent((PanelClickedEventHandler) e ->
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
+        ui.registerEvent((ShowRenamePanelEventHandler) e -> Platform.runLater(() ->
+                ((FilterPanel) getPanel(e.panelId)).showRenameTextField()));
```

Comment:
```ShowRenamePanelEvent``` is called by ```UseGlobalConfigsTest``` and ```PanelRenameTest``` as well. Removing ```runLater``` resulted in those tests failing to show the rename text field.